source/manage_project/create_new_project.md

```diff
@@ -0,0 +1,37 @@
+# Create a new project
+
+1. Open the command line interface.
+
+1. Navigate to your documents folder.
+
+1. Run the following in the command line:
+
+    ```
+    Middleman init PROJECT_NAME -T alphagov/tech-docs-template
```

Comment:
Not a problem with this PR but as I mentioned elsewhere, we should find a standard for placeholders.